com.github.LongSoft.UEFITool.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.github.LongSoft.UEFITool
+runtime: org.kde.Platform
+runtime-version: "6.3"
+sdk: org.kde.Sdk
+rename-desktop-file: uefitool.desktop
+rename-icon: uefitool
+command: uefitool
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
```

Comment:
Why is this needed?

Reply:
`--share=ipc` and `--filesystem=home` have been removed after further testing.